.github/workflows/sh-checker.yml

```diff
@@ -0,0 +1,17 @@
+name: SH Checker
+on:
+  - pull_request
+jobs:
+  sh-checker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run the sh-checker
+        uses: luizm/action-sh-checker@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SHELLCHECK_OPTS: -o all -e 2148 -e SC2033 -e SC2032 -e SC2034 -e SC2154 -e SC2312 -e SC2164
+          SHFMT_OPTS: -s -i 2 -bn -ci -sr -s
+        with:
+          sh_checker_comment: true
+          sh_checker_exclude: "packages/emacs/emacs.pacscript packages/emacs-git/emacs-git.pacscript"
```

Comment:
is this list going to grow ever longer?

Reply:
Unless there is a specific request, no. We only added it because Wizard wanted some comments to be visually aligned.